### PR TITLE
Network crash fixes

### PR DESCRIPTION
**Pull request by Plootie**

### CODE REVIEW

File: SIT.Manager/Localization/zh-TW.axaml [hunk 1]
```diff
@@ -444,7 +444,7 @@
 	<system:String x:Key="DirectConnectViewEnterPasswordPlaceholder">Enter Password...</system:String>
 	<system:String x:Key="DirectConnectViewRememberMeCheckBoxTitle">Remember Me</system:String>
 	<system:String x:Key="DirectConnectViewConnectButtonTitle">Connect</system:String>
-	<system:String x:Key="CDirectConnectViewonnectButtonTitleToolTip">Connect to the server</system:String>
+	<system:String x:Key="DirectConnectViewonnectButtonTitleToolTip">Connect to the server</system:String>
```
Comment (artehe) [inline SIT.Manager/Localization/zh-TW.axaml:447]: I don't know how the C ended up at the front 😆 

Can I be a pain and ask you to make the key DirectConnectViewConnectButtonTitleToolTip and change it in the other localisations and the DirectConnectView please?

**Review by artehe — APPROVED** [review]
Just the one comment I had and the conflicts to resolve then I think this should be good to go 👍

File: SIT.Manager/Localization/zh-TW.axaml (hunk 1 above)
Comment (Plootie) [reply SIT.Manager/Localization/zh-TW.axaml:447]: Sure np

Comment (J0nathan550) [reply SIT.Manager/Localization/zh-TW.axaml]: eep

**Pull request merged**